teuthology/suite/run.py

```diff
@@ -175,7 +185,10 @@ def choose_ceph_hash(self):
         tip.
         """
         repo_name = self.ceph_repo_name
-
+        if not repo_name.endswith('.git'):
+            raise ValueError("Invalid repo name format. Must end with '.git'")
+        if not repo_name.startswith('https://github.com'):
+            raise ValueError("Invalid repo name format. Must start with 'https://github.com'")
```

Comment:
I know this was added because of my mistake while testing... but I'm not sure if we should be adding a check for 'https://github.com', because it could be "git.ceph.com" or gitlab or potentially something else? 

Also, `--ceph-repo https://github.com/ceph/ceph` (without ".git" at the end) is a valid argument of teuthology CLI right now. So this could be a breaking change.
I think we can instead add this check in teuthology-api or pulpito-ng to make it beginner friendly? What do you think?

Reply:
Oh, yeah, we don't want to be limited to just github.com, or even git.ceph.com. And ending with ".git" shouldn't be a requirement.

Reply:
you guys are correct, removed this part.